### DISCUSSION TITLE
HEIC support for iOS?

### DISCUSSION THREAD

**Discussion by estellise-yukihime** [yasirkula/UnityNativeGallery#169] [Q&A] [answered]
Does this plugin has support for HEIC file format for iOS?

**Comment by yasirkula** [comment, marked as answer]
Yes, you can load HEIC images with NativeGallery.LoadImageAtPath.

**Reply by estellise-yukihime** [reply]
I've tried testing it in my MacOS Catalina Version 10.15, but it seems to be grayed out, and I cant select a HEIC file, is this behavior okay because it is a Mac and not an iOS device?

![image](https://user-images.githubusercontent.com/44573967/108837531-55bc7a80-760d-11eb-89d4-14578eaaaa8e.png)

**Reply by yasirkula** [reply]
This behaviour is expected, HEIC will work on iOS only.
